src/main/java/me/totalfreedom/totalfreedommod/command/Command_rainbowtag.java

```diff
@@ -0,0 +1,49 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import java.util.Iterator;
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import me.totalfreedom.totalfreedommod.util.FUtil;
+import org.apache.commons.lang3.StringUtils;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = Rank.OP, source = SourceType.ONLY_IN_GAME)
+@CommandParameters(description = "Gives you a rainbow tag", usage = "/<command> <tag>")
+public class Command_rainbowtag extends FreedomCommand
+{
+
+    @Override
+    public boolean run(CommandSender sender, Player playerSender, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        if (args.length < 1)
+        {
+            return false;
+        }
+      
+        final String tag = ChatColor.stripColor(FUtil.colorize(StringUtils.join(args, " ")));
+        
+        if(tag.length() > 20)
+        {
+            msg("That tag is too long (Max is 20 characters).");
+            return true;
+        }
+        
+        for (String word : Command_tag.FORBIDDEN_WORDS) 

```

Comment:
This will need to be changed, assuming there is a merge of the 5.0.2 release as the forbidden tags system has changed. That's something I can change though once the PR goes in.